test/controllers/PensionDetailsSconControllerSpec.scala

```diff
@@ -55,7 +55,7 @@ class PensionDetailsControllerSpec extends PlaySpec with GuiceOneServerPerSuite
   implicit val messagesProvider=MessagesImpl(Lang("en"), messagesAPI)
   implicit val ac=app.injector.instanceOf[ApplicationConfig]
   implicit val gmpSessionCache=app.injector.instanceOf[GmpSessionCache]
-  lazy val pensionDetailsForm = new PensionDetailsForm(mcc)
+  lazy val pensionDetailsForm = new PensionDetails_no_longer_used_Form(mcc)
```

Comment:
can we use the new form created?  PensionDetailsSconForm?

Reply:
This journey doesn't deal with this form. Will be picked later